docs/src/main/paradox/durable-state.md

```diff
@@ -0,0 +1,48 @@
+# State changes from Akka Persistence State
+
+A typical source for Projections is changes stored with @apidoc[DurableStateBehavior$] in [Akka Persistence](https://doc.akka.io/docs/akka/current/typed/persistence-durable-state.html). Durable state changes can be [tagged](https://doc.akka.io/docs/akka/current/typed/persistence-durable-state.html#tagging) and then
+consumed with the [changes query](https://doc.akka.io/docs/akka/current/persistence-query-durable-state.html#using-query-with-akka-projections).
+
+Akka Projections has integration with `changes`, which is described here. 
+
+## Dependencies
+
+To use the Durable State module of Akka Projections add the following dependency in your project:
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.lightbend.akka
+  artifact=akka-projection-state_$scala.binary.version$
+  version=$project.version$
+}
+
+Akka Projections require Akka $akka.version$ or later, see @ref:[Akka version](overview.md#akka-version).
+
+@@project-info{ projectId="akka-projection-state" }
+
+### Transitive dependencies
+
+The table below shows `akka-projection-state`'s direct dependencies and the second tab shows all libraries it depends on transitively.
+
+@@dependencies{ projectId="akka-projection-state" }
+
+## SourceProvider for changesByTag
+
+A @apidoc[SourceProvider] defines the source of the event envelopes that the `Projection` will process. A `SourceProvider`
+for the `changes` query can be defined with the @apidoc[DurableStateStoreProvider$] like this:
+
```

Comment:
The naming of `changes` (in the Persistence API)  VS `changesByTag` in the projections API is so confusing... </rant>

Reply:
+1